nodejs/examples/vpc/index.ts

```diff
@@ -55,12 +56,23 @@ const vpcWithLocations = new awsx.ec2.Vpc("custom6", {
     ],
 }, providerOpts);
 
-const vpcWithSpecificZones = new awsx.ec2.Vpc("custom7", {
-    requestedAvailabilityZones: ["us-east-1a", "us-east-1b", "us-east-1c"],
-    subnets: [
-        { type: "public" },
-        { type: "private" },
-        { type: "isolated", name: "db" },
-        { type: "isolated", name: "redis" },
-    ],
-}, { provider: new aws.Provider("prov2", { region: "us-east-1" }) });
+
+export = async () => {
```

Comment:
@leezen Since I am using the existing provider, and I don't know where that's located as it's an ENV variable on the CI, I am fetching all the zones and that seems to be a promise, so I looked around and this was used in the ELB tests, so I assumed it'll be fine?

Reply:
@leezen Actually, I simplified by using promises directly as arguments to the `random.Shuffle` in https://github.com/pulumi/pulumi-awsx/pull/638/commits/da138a4efea0bc1a5e028f16c18f1b7c7380d8c1 - let me know how you prefer it, or if you want me to go back to using hard coded region and zones  in a new custom provider with a different name - since (I am guessing) the name was the reason it failed?

Reply:
Yeah, this makes sense if you want to just fetch the available zones and shuffle them in. That works for me.